ftl/core/search.ftl

```diff
@@ -16,7 +16,7 @@ search-unclosed-quote = an opening double quote `"` was found, but there was no
 search-missing-key = a colon `:` was found, but there was no keyword preceding it. If you want to search for a literal `:`, prepend a backslash: `\:`.
 search-unknown-escape = the escape sequence `{ $val }` is not defined. If you want to search for a literal backslash `\`, prepend another one: `\\`.
 search-invalid-argument = `{ $term }` was given an invalid argument '`{ $argument }`'.
-search-invalid-flag = `flag:` must be followed by a valid flag number: `1` (red), `2` (orange), `3` (green), `4` (blue) or `0` (no flag).
+search-invalid-flag = `flag:` must be followed by a valid flag number: `1` (red), `2` (orange), `3` (green), `4` (blue), `5` (pink), `6` (turquoise), `7` (purple) or `0` (no flag).
```

Comment:
we'll need to create a new string key for this, as AFAIK simply changing the text will not notify translators that the string needs changing. search-invalid-flag-2 or similar would be fine

Reply:
Isn't there a way to mark a string as updated? I think this is a good example where it would make more sense to fall back on an outdated translation than on another language.

Reply:
Pity, but thanks for explaining.